pagopa/io-app
# feat: [IOBP-437] Add new wallet payment outcome error handling

⚠️ Depends on https://github.com/pagopa/io-app/pull/5378

## Short description
This PR adds the handling of the payment error outcomes.

## List of changes proposed in this pull request
- Added components mapping based on the outcome in `WalletPaymentOutcomeScreen`
- Added required locales keys
- Refactored `usePaymentFailureSupportModal` to remove the requirement of passing `RptId` as its argument.

## How to test
Within the **Profile > New Wallet > Payment** playground, using the `io-dev-api-server`, try to simulate different payment outcomes and check that the screen and its components are displayed correctly.

## Preview

https://github.com/pagopa/io-app/assets/6160324/cb7c33f1-9c0d-471c-accf-a362efc42264
